### PR TITLE
LPS-166242 [EVP] Create buttons according to status

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-166242](https://issues.liferay.com/browse/LPS-166242)

### CODE REVIEW

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by EuniceTravassos** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/button-change-status/fragment.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"icon": "button",
+	"jsPath": "index.js",
+	"name": "Button Change Status",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/button-change-status/fragment.json:7]: Please rename this to "Organization Status Update Buttons - Staff". Also rename the folder: "organization-status-update-buttons-staff"
```suggestion
	"name": "Organization Status Update Buttons - Staff",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-organization-details/page-definition.json [hunk 2]
```diff
@@ -31,32 +86,209 @@
...
+																			"id": "tablet"
+																		}
+																	],
+																	"size": 2
+																},
+																"pageElements": [
+																	{
+																		"definition": {
+																			"fragment": {
+																				"key": "button-change-status"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-organization-details/page-definition.json:248]: As soon as you change the fragment's folder name, substitute here as well
```suggestion
																				"key": "organization-status-update-buttons-staff"
```

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-organization-details/page-definition.json (hunk 2 above)
Comment (juliocezar96) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-organization-details/page-definition.json]: Adjusted !

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by Claudionilton** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @juliocezar96

**Review by EuniceTravassos — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/organization-status-update-buttons-staff/index.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/organization-status-update-buttons-staff/index.js:1]: Why do you use it? Is it necessary?

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Review by silasbsi — APPROVED** [review]

**Pull request closed without merging**